.github/workflows/testing-alpine.yml

```diff
@@ -20,6 +20,8 @@ jobs:
     steps:
       - run: apk add libc-dev gcc make automake autoconf pkgconf python3 file diffutils git
 
+      - run: git config --global --add safe.directory /__w/ctags/ctags
+
```

Comment:
What's this?
Is this something to do with BSD make?
I guess this is not for BSD make. Just for fixing an issue building on alpine.
If my guessing is correct, could you consider a pull  request only for alpine?
So, we can merge it quickly. This fix is needed to make #3350 forward.

Reply:
yes, you are right. I will file a new PR.